src/versionbits.h

```diff
@@ -53,6 +61,7 @@ class AbstractThresholdConditionChecker {
 public:
     // Note that the functions below take a pindexPrev as input: they compute information for block B based on its parent.
```

Comment:
Yeah ok this makes sense to me. I'll take my function out of the "functions below" list and rename variable to `pindex`. Should correcting `GetStateSinceHeightFor()` happen in a different commit though?